artillery-core/src/epidemic/cluster.rs

```diff
@@ -39,6 +39,10 @@ impl Cluster {
             .unwrap();
     }
 
+    pub fn send_payload(&self, id: Uuid, msg: String) {
+        self.comm.send(ArtilleryClusterRequest::Payload(id, msg));
```

Comment:
Shouldn't this take something like [`bytes::Bytes`][] instead of `String` ?

[`bytes::Bytes`]: https://docs.rs/bytes/0.5.4/bytes/struct.Bytes.html

Reply:
No, `bytes` crate is useless for our case. But I am ok to have a variant which takes a slice.